testapp1.py

```diff
@@ -1,68 +1,83 @@
 import os
 import uuid
 from datetime import timedelta
-from flask import Flask, redirect, request, session, url_for
+
 import jwt
+from quart import Quart, redirect, request, session, url_for
+
 from generate_jwt import generate_jwt
 
-app = Flask(__name__)
+app = Quart(__name__)
 
 # Generate a secret key for the session cookie
-SECRET_KEY = os.getenv('SECRET_KEY', str(uuid.uuid4()))
+SECRET_KEY = os.getenv("MEMBRANE_SECRET_KEY", str(uuid.uuid4()))
 app.secret_key = SECRET_KEY
 
-session_lifetime_minutes = int(os.environ.get('SESSION_LIFETIME_MINUTES', 30))
-app.config['PERMANENT_SESSION_LIFETIME'] = timedelta(minutes=session_lifetime_minutes)
+session_lifetime_minutes = int(os.getenv("MEMBRANE_SESSION_LIFETIME_SECONDS", 30))
```

Comment:
you're putting seconds values into a _minutes variables...

Reply:
done and fixed everywhere